src/multilevel_bisect/coarsen.cpp

```diff
@@ -323,12 +323,14 @@ pmondriaan::hypergraph coarsen_hypergraph_seq(bulk::world& world,
                                               pmondriaan::contraction& C,
                                               pmondriaan::options& opts,
                                               std::mt19937& rng) {
+    simplify_duplicate_nets(H);

```

Comment:
This gives problems on a distributed hypergraph!

Reply:
I thought I worked out every bug around this method, since it is only used in sequential phases. What seems to be the problem?

Reply:
I missed that it was in the sequential method :)